docs/docs/dependencies/index.md

```diff
@@ -153,11 +228,13 @@ export const Page: DocumentInterface<Pick<GQLPage, "content" | "seo">, GQLPageIn
 };
 ```
 
-### 3. Register the DependencyInterface at the DependenciesConfigProvider
+#### 3. Admin: Register the `DependencyInterface` at the `DependenciesConfigProvider`
+
+This allows you to access the `displayName` and `resolvePath` method via the `useDependenciesConfig()` hook.
```

Comment:
Is this something one would need to do?

Reply:
No. It was just to "build a mental bridge" between steps 2 and 3 so readers can understand how the methods are used.

But I understand your point. Within this guide it can be confusing because readers might think they must use it somewhere. So I removed it in 16ab20ca797efb3e92fcb738aab8f758af600f8f